pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -60,6 +61,9 @@ func RunClusterInfo(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command) error
 		return err
 	}
 	printService(out, "Kubernetes master", client.Host)
+	if len(client.AlternateHosts) > 0 {
```

Comment:
Does not have to be merged, just also displayed at minimum.


On Aug 22, 2016, at 8:59 AM, Dmitry Shulyak <notifications@github.com>
wrote:

> @@ -60,6 +61,9 @@ func RunClusterInfo(f *cmdutil.Factory, out io.Writer,
cmd *cobra.Command) error

>  		return err
>  	}
>  	printService(out, "Kubernetes master", client.Host)
> +	if len(client.AlternateHosts) > 0 {

you mean that AlternateHosts should be merged with Servers when user
executes kubectl config view?